Mailer/app/api/v1/Handlers/Users/SubscribeHandler.php

```diff
@@ -67,68 +68,21 @@ public function handle($params)
         return new JsonApiResponse(200, ['status' => 'ok']);
     }
 
-    /**
-     * Validate and load email from $params
-     *
-     * @param $params
-     * @return string
-     * @throws InvalidApiInputParamException
-     */
-    protected function getUserEmail($params): string
```

Comment:
By removing this function,  `BulkSubscribeHandler` implementation broke. Please make necessary changes to make it work again.